src/ServiceControl.Monitoring/Metrics/Raw/OccurrenceSerializerDefinition.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace ServiceControl.Monitoring.Metrics.Raw
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using NServiceBus.MessageInterfaces;
+    using NServiceBus.Serialization;
+    using NServiceBus.Settings;
+
+    public class OccurrenceSerializerDefinition : SerializationDefinition
+    {
+        public override Func<IMessageMapper, IMessageSerializer> Configure(ReadOnlySettings settings)
+        {
+            return mapper => new OccurrenceSerializer();
+        }
+    }
+
+    public class OccurrenceSerializer : RawMessageSerializer<Occurrences>, IMessageSerializer
+    {
+        public object[] Deserialize(Stream stream, IList<Type> messageTypes = null)
+        {
+            long baseTicks = 0;
```

Comment:
It turned out to be simpler to just drop the timestamp management into the base class. Both current implementation manage timestamps the same way. 

https://github.com/Particular/ServiceControl.Monitoring/pull/38